README.md

```diff
@@ -1,2 +1,96 @@
-# thunder
-Asgardeo Thunder is a Go based Identity and Access Management product
+# ⚡ Asgardeo Thunder — Cloud-Native Identity Management
+
+**Asgardeo Thunder** is a modern, cloud-native identity management service. It empowers you to design tailored login, registration, and recovery flows using a flexible identity flow designer.
+
+Thunder secures users, applications, services, and AI agents by managing their identities and offering a complete suite of supporting capabilities.
+
+Designed for extensibility, scalability, and seamless containerized deployment, Thunder integrates naturally with microservices and DevOps environments—serving as the core identity layer for your cloud platform.
+
+---
+
+## 🚀 Features
+
+- ✅ **Standards-Based**
+  - OAuth 2.1, OpenID Connect (OIDC)
+  - SCIM 2.0
+- 🛠️ **Visual Identity Flow Designer**
+- 👤 **User & Identity Management**
+- 🔗 **Social Login**
+- 🔐 **Multi-Factor Authentication (MFA)**
+- 🌐 **RESTful APIs**
+
+---
+
+## ⚡ Quickstart
+
+### ✅ Prerequisites
+
+- Go 1.23+
+- Docker
+- cURL
+
+---
+
+### 🔧 Step 1: Start PostgreSQL
+
+```bash
+docker run -d -p 5432:5432 --name postgres \
+  -e POSTGRES_USER=asgthunder \
+  -e POSTGRES_PASSWORD=asgthunder \
+  -e POSTGRES_DB=thunderdb \
+  postgres
+```
+
+### 🗂 Step 2: Initialize the Database
+
+```bash
+docker exec -i postgres psql -U asgthunder -d thunderdb < dbscripts/schema.sql
+```
+
+---
+
+### 🛠 Step 3: Build the Product
+
+```bash
+make all
+```
+
+---
+
+### ▶️ Step 4: Run the Product
+
+```bash
+cd target
+unzip thunder-1.0.0-m1-SNAPSHOT.zip
```

Comment:
Better not to follow java snapshot release semantics IMO.

Reply:
We will revisit the versioning strategy entirely once the UI components are integrated. Till then, we can things simple, so proceeding with this.